codecept.conf.js

```diff
@@ -64,14 +64,11 @@ exports.config = {
     },
     GenerateReportHelper: {
       require: './e2e/helpers/generate_report_helper.js',
-    },
-    TabAssertions: {
-      require: './e2e/helpers/tab_assertions_helper.js',
-    },
+    }
   },
   include: {
     config: './e2e/config.js',
-    I: './e2e/steps_file.js',
+    I: './e2e/actors/main.js',
```

Comment:
I imagine you'll need to do the same for the crossbrowser tests in `saucelabs.conf.js`

Reply:
good spot, was not aware about this file
done